index.html

```diff
@@ -1,13 +1,250 @@
+                <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                  <span class="info-text">2015</span></li>
+              </ul>
+        
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+          
+          <ul class="tech-list">
+            <li class="project-tech">html</li>
+            <li class="project-tech">css</li>
+            <li class="project-tech">javascript</li>
+          </ul>
+            <button class="outlined-button">See Project</button>
+        
+        </div>
+    </div>
+</section>
+<section class="portfolio">
+  <div class="project">
+      <img src="./assets/tonic-girl.png" alt="" class="project-preview">
+
+      <div class="project-info">
+          <h2 class="h-text project-name">Tonic</h2>
+            <ul class="skills-list">
+              <li class="checkbox"><span class="info-text">CANOPY</span></li>
+              <li class="checkbox"><i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">Back End Dev</span></li>
+              <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">2015</span></li>
+            </ul>
+        
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        <ul class="tech-list">
+          <li class="project-tech">html</li>
+          <li class="project-tech">css</li>
+          <li class="project-tech">javascript</li>
+        </ul>
+          <button class="outlined-button">See Project</button>
+      </div>
+  </div>
+</section>
+
+<section class="portfolio test2">
+  <div class="project">
+      <img src="./assets/multips.png" alt="" class="project-preview">
+
+      <div class="project-info">
+          <h2 class="h-text project-name">Tonic</h2>
+            <ul class="skills-list">
+              <li class="checkbox"><span class="info-text">CANOPY</span></li>
+              <li class="checkbox"><i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">Back End Dev</span></li>
+              <li class="checkbox"> <i class="ri-checkbox-blank-circle-fill dots"></i>
+                <span class="info-text">2015</span></li>
+            </ul>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        <ul class="tech-list">
+          <li class="project-tech">html</li>
+          <li class="project-tech">css</li>
+          <li class="project-tech">javascript</li>
+        </ul>
+          <button class="outlined-button">See Project</button>
+      </div>
+  </div>
+</section>
+<section class="about">
+      <div class="about-intro-text">
+          <h1 class="project-name h-text1">About <br> My self</h1>
```

Comment:
- To help improve the SEO of your site, it is advisable to use only one `h1` tag on your site. Please, just as the previous reviewers advised please use an `hx` tag between `h2-h6`. Please fix this 🙏